docs/contrib/Metrics.rst

```diff
@@ -0,0 +1,48 @@
+Metrics
+=======
+
+The `OpenROAD-flow-scripts`_ repository contains source files (e.g., LEF/DEF,
+verilog, SDC, Liberty, RC) and configuration files (e.g. `config.mk`)
+that enable the user to run a small set of designs through our complete
+RTL-GDS flow. To keep track of the quality of the results, we maintain
+inside each design folder two files: (i) `metadata-base-ok.json` and (ii)
+`rules.json`. The file (i) contains all the relevant information extracted
+from the "golden" execution of the flow (i.e., last known good result). On
+the other hand file (ii) holds a set of rules that we use to evaluate new
+executions when a change is made.  The evaluation checks for key metrics
```

Comment:
Avoiding splitting this:

... two files:
(i) `metadata-base-ok.json`  which contains all the relevant information extracted
from the "golden" execution of the flow (i.e., last known good result)
(ii) `rules.json` which holds a set of rules that we use to evaluate new
executions when a change is made

Reply:
Are metrics updates done only by developers or are users allowed to do so?

Reply:
@dralabeing Anyone making changes with an impact on metrics should submit such a PR.  The current team will have to manage the private PDK related metrics.

Reply:
OK

Reply:
Fixed on 3a3c888